src/crl.rs

```diff
@@ -87,13 +86,12 @@ impl CertRevocationList for OwnedCertRevocationList {
         &self,
         supported_sig_algs: &[&SignatureAlgorithm],
         issuer_spki: &[u8],
-        budget: &mut Budget,
     ) -> Result<(), Error> {
         signed_data::verify_signed_data(
             supported_sig_algs,
             untrusted::Input::from(issuer_spki),
             &self.signed_data.borrow(),
-            budget,
+            &mut Budget::default(),
```

Comment:
This feels like an acceptable crate-internal workaround for a point release to undo the mistake. I'll commit to a better solution in https://github.com/rustls/webpki/pull/186